doc/rados/configuration/journal-ref.rst

```diff
@@ -1,7 +1,7 @@
 ==========================
  Journal Config Reference
 ==========================
-
+.. warning:: Filestore removed since Reef release and is no longer supported.
```

Comment:
Inconsistency: lacks _is_. Beside that please see the Neha's comment.

Reply:
Actually, regarding the Journal and other parts of Filestore, is it worth to keep that doc at all?

Reply:
Also the filestore-config-ref.rst, can we remove it ?

Reply:
We should remove specific references to Filestore in Reef and main, but not backport to Quincy and earlier where it is still supported.